DSharpPlus/Entities/DiscordChannel.cs

```diff
@@ -160,8 +210,14 @@ internal DiscordChannel()
         /// <param name="tts">Whether the message is to be read using TTS.</param>
         /// <param name="embed">Embed to attach to the message.</param>
         /// <returns>The sent message.</returns>
-        public Task<DiscordMessage> SendMultipleFilesAsync(Dictionary<string, Stream> files, string content = "", bool tts = false, DiscordEmbed embed = null) =>
-            this.Discord.ApiClient.UploadFilesAsync(Id, files, content, tts, embed);
+        public Task<DiscordMessage> SendMultipleFilesAsync(Dictionary<string, Stream> files, string content = "", bool tts = false, DiscordEmbed embed = null)
+        {
+            if (!this.Type.IsText())
```

Comment:
I think it'd be better to just `this.Type != ChannelType.Voice && this.Type != ChannelType.Category`